src/riskmatrix/views/risk_assessment.py

```diff
@@ -296,112 +321,115 @@ def __html__(self) -> str:
-        ],
-        [
-
-            Cell(value='Likelihood', header=True, colspan=7),
-        ]
-    ]
-
-    session = request.dbsession
-    query = session.query(RiskAssessment)
-    query = query.filter(RiskAssessment.organization_id == context.id)
-    query = query.join(RiskAssessment.asset)
-    query = query.join(RiskAssessment.risk)
-
-    assessments = []
-    index = 0
-    for assessment in query:
-        impact = assessment.impact
-        likelihood = assessment.likelihood
-        if impact and likelihood:
-            index += 1
-            assessments.append({
-                'nr': index + 1,
-                'name': assessment.risk.name,
-                'asset': assessment.asset.name,
-                'impact': impact,
-                'likelihood': likelihood,
-            })
-
-            severity = 'success'
-            if impact * likelihood >= 5:
-                severity = 'warning'
-            if impact * likelihood >= 15:
-                severity = 'danger'
-
-            row = 5 - impact
-            col = likelihood
-            if row == 0:
-                col += 1
-            cells[row][col].value += Markup(
-                ' <span class="badge rounded-pill bg-{severity} {css_class}"'
-                ' title="{title}">{nr}</span>'
-            ).format(
-                severity=severity,
-                nr=index,
-                title=f'{assessment.asset.name}: {assessment.risk.name}',
-                css_class='text-dark' if severity == 'warning' else ''
+    # Create a 5x5 grid and set the color for each cell
+    for color, indices in colors.items():
+        for index in indices:
+            i, j = divmod(index, 5)
+
+            fig.add_shape(
+                type="rect",
+                x0=j,
+                y0=4 - i,
+                x1=j + 1,
+                y1=5 - i,
+                line={"color": "white", "width": 0},
+                fillcolor=color,
+                layer="below",
             )
 
+    # Plot points
+    for risk in list(risks):
+        if risk.likelihood and risk.impact:
+            x, y = float(risk.likelihood - 1), float(4 - (risk.impact - 1))
+
+            # Adjust the position within the cell to avoid overlap
+            noise_x = np.random.uniform(0.1, 0.9)
+            noise_y = np.random.uniform(0.1, 0.9)
+            x, y = x + noise_x, y + noise_y
+
+            fig.add_trace(
+                go.Scatter(
+                    x=[x],
+                    y=[y],
+                    text=[str(risk.nr)],
+                    name="",
+                    mode="markers+text",
+                    marker={"color": "black", "size": 18},
+                    textposition="middle center",
+                    hoverinfo="text",
+                    hovertemplate=f"{risk.nr} {risk.name} \
+                    (Impact: {risk.impact} Likelihood: {risk.likelihood})",
+                    textfont={"color": "white"},
```

Comment:
```suggestion
                    name='',
                    mode='markers+text',
                    marker={'color': 'black', 'size': 18},
                    textposition='middle center',
                    hoverinfo='text',
                    hovertemplate=f'{risk.nr} {risk.name} '
                    f'(Impact: {risk.impact} Likelihood: {risk.likelihood})',
                    textfont={'color': 'white'},
```
style nit. Don't use broken lines, multiple strings separated by white space auto concatenate.